### PR TITLE
chore: Remove old header formatting equal signs

### PR DESCRIPTION
**Pull request by nschonni**
Not sure if this is in other files, just saw this one reading the article

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/upgrade-freemium.md [hunk 1]
```diff
@@ -22,7 +22,6 @@ appliesto:
 ---
 
 # Upgrade Microsoft Teams free to subscription version
-======================================================
 
 If your organization is using the free version of Microsoft Teams, you can easily upgrade to the full version by purchasing a Microsoft 365 or Office 365 subscription plan for your users. The full version provides additional Teams features — like scheduling, audio conferencing, enhanced administration, and security capabilities — that the free version doesn't provide. Microsoft 365 and Office 365 combine the familiar Microsoft Office desktop suite with cloud-based versions of Microsoft's next-generation communications and collaboration services — including Exchange Online, SharePoint Online, and Office — to help users be productive from virtually anywhere through the internet. When you upgrade Teams, your existing Teams data will not be lost; all your teams, channels, chats, files, and permissions come with you.
```
Comment (JohanFreelancer9) [inline Teams/upgrade-freemium.md:26]:
```suggestion
If your organization is using the free version of Microsoft Teams, you can easily upgrade to the full version by purchasing a Microsoft 365 or Office 365 subscription plan for your users. The full version provides additional Teams features—like scheduling, audio conferencing, enhanced administration, and security capabilities—that the free version doesn't provide. Microsoft 365 and Office 365 combine the familiar Microsoft Office desktop suite with cloud-based versions of Microsoft's next-generation communications and collaboration services—including Exchange Online, SharePoint Online, and Office—to help users be productive from virtually anywhere through the internet. When you upgrade Teams, your existing Teams data will not be lost; all your teams, channels, chats, files, and permissions come with you. 
```

Comment (JohanFreelancer9) [reply Teams/upgrade-freemium.md:26]: Please remove the spaces before and after the four em dashes.

Comment (nschonni) [reply Teams/upgrade-freemium.md:26]: These changes are unrelated to this PR. Personally, I thinking making them look like hyphenated words makes it harder to read

**Comment by scanum** [comment]
@LolaJacobsen  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

File: Teams/upgrade-freemium.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/upgrade-freemium.md:26]: @nschonni As you rightly say, these changes are unrelated to this PR. It is not a problem, we can leave them as is, but my edits were made in accordance with the Microsoft Style Guide, to quote: "Use an em dash (—) to set off a parenthetical phrase with more emphasis than parentheses provide. Don’t add spaces around an em dash."

**Pull request merged**